docs/modules/ROOT/pages/index.adoc

```diff
@@ -46,7 +46,7 @@ xref:module/access-control.adoc[[.card-title]#Access Control# [.card-body]#pass:
 == Available networks
 Defender works with most mainnet and testnet networks, as well as local mainnet forks.
 
-- https://ethereum.org/en/[*Ethereum Mainnet*, window=_blank], *Goerli* testnet and *Sepolia* testnet.
+- https://ethereum.org/en/[*Ethereum Mainnet*, window=_blank], *Goerli* testnet, *Sepolia* testnet and *Holesky* testnet.
```

Comment:
Should we remove Goerli given it is deprecated?

Reply:
Good point, lemme do that